commit 2efc547c5e535a2061312e63a459cdee9635cdf9
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Mon Mar 7 13:38:39 2016 +0000

    Pi3 DT: Add dtparams for the SD interface
    
    Add new base dtparams sd_overclock, sd_force_pio, sd_pio_limit
    and sd_debug. These were missed out of the initial Pi3 DTB.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
index cc06089..36972d8 100644
--- a/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
+++ b/arch/arm/boot/dts/bcm2710-rpi-3-b.dts
@@ -188,5 +188,9 @@
 		audio = <&audio>,"status";
 		watchdog = <&watchdog>,"status";
 		random = <&random>,"status";
+		sd_overclock = <&sdhost>,"brcm,overclock-50:0";
+		sd_force_pio = <&sdhost>,"brcm,force-pio?";
+		sd_pio_limit = <&sdhost>,"brcm,pio-limit:0";
+		sd_debug     = <&sdhost>,"brcm,debug";
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1165_2efc547c5e535a2061312e63a459cdee9635cdf9.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1165_2efc547c5e535a2061312e63a459cdee9635cdf9.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
